.github/workflows/release.yml

```
@@ -61,7 +61,7 @@ jobs:
       - name: Install cibuildwheels
         run: python -m pip install cibuildwheel==2.20.0
       - name: Build ittapi python wheels
-        run: python -m cibuildwheel python --output-dir python_dist/${{ runner.os }}
+        run: python -m cibuildwheel python --output-dir python_dist/
```

```suggestion
        run: python -m cibuildwheel python --output-dir python_dist
```
make it more consistent

---

Done.